meilidb-http/src/routes/index.rs

```diff
@@ -12,104 +17,313 @@ use crate::models::token::ACL::*;
+                    .map_err(ResponseError::internal)?
+                    .ok_or(ResponseError::internal("'created_at' date not found"))?;
+                let updated_at = index
+                    .main
+                    .updated_at(&reader)
+                    .map_err(ResponseError::internal)?
+                    .ok_or(ResponseError::internal("'updated_at' date not found"))?;
+
+                let index_reponse = IndexResponse {
+                    name,
+                    uid: index_uid,
+                    created_at,
+                    updated_at,
+                };
+                response_body.push(index_reponse);
+            }
+            None => error!(
+                "Index {} is referenced on list of indexes but cannot be found",
+                index_uid
+            ),
+        }
+    }
+
+    Ok(tide::response::json(response_body))
 }
 
-pub async fn get_index_schema(ctx: Context<Data>) -> SResult<Response> {
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
+struct IndexResponse {
+    name: String,
+    uid: String,
+    created_at: DateTime<Utc>,
+    updated_at: DateTime<Utc>,
+}
+
+pub async fn get_index(ctx: Context<Data>) -> SResult<Response> {
     ctx.is_allowed(IndexesRead)?;
 
     let index = ctx.index()?;
 
     let env = &ctx.state().db.env;
     let reader = env.read_txn().map_err(ResponseError::internal)?;
 
-    let schema = index
+    let uid = ctx.url_param("index")?.to_string();
+    let name = index
         .main
-        .schema(&reader)
-        .map_err(ResponseError::create_index)?;
+        .name(&reader)
+        .map_err(ResponseError::internal)?
+        .ok_or(ResponseError::internal("'name' not found"))?;
+    let created_at = index
+        .main
+        .created_at(&reader)
+        .map_err(ResponseError::internal)?
+        .ok_or(ResponseError::internal("'created_at' date not found"))?;
+    let updated_at = index
+        .main
+        .updated_at(&reader)
+        .map_err(ResponseError::internal)?
+        .ok_or(ResponseError::internal("'updated_at' date not found"))?;
+
+    let response_body = IndexResponse {
+        name,
+        uid,
+        created_at,
+        updated_at,
+    };
 
-    match schema {
-        Some(schema) => {
-            let schema = SchemaBody::from(schema);
-            Ok(tide::response::json(schema))
-        }
-        None => Ok(
-            tide::response::json(json!({ "message": "missing index schema" }))
-                .with_status(StatusCode::NOT_FOUND)
-                .into_response(),
-        ),
-    }
+    Ok(tide::response::json(response_body))
+}
+
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+struct IndexCreateRequest {
+    name: String,
+    schema: Option<SchemaBody>,
+}
+
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
+struct IndexCreateResponse {
+    name: String,
+    uid: String,
+    schema: Option<SchemaBody>,
```

Comment:
Do you prefer having a `schema: null` in the response body? If not apply this change :)
```suggestion
    #[serde(skip_serializing_if = "Option::is_none")]
    schema: Option<SchemaBody>,
```